core/vm/gas_table.go

```diff
@@ -95,8 +95,9 @@ var (
 
 func gasSStore(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize uint64) (uint64, error) {
 	var (
-		y, x         = stack.Back(1), stack.Back(0)
-		current, err = evm.StateDB.GetState(contract.Address(), x.Bytes32())
+		y, x                      = stack.Back(1), stack.Back(0)
+		internalContractAddr, err = contract.Address().InternalAddress()
+		current, _                = evm.StateDB.GetState(*internalContractAddr, x.Bytes32())
```

Comment:
will this not return an error ?

Reply:
https://github.com/dominant-strategies/go-quai/pull/657/commits/cd76687c6060f9f9965e69237885eb6221754558 reverts this so no error is returned